content/logs/live_tail.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Live Tail
+kind: documentation
+description: "See all your logs in real time."
+further_reading:
+- link: "logs/graph"
+  tag: "Documentation"
+  text: "Perform analytics with Log Graphs"
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+{{< img src="logs/live_tail/live_tail_demo.gif" alt="Live tail" responsive="true" popup="true">}}
+
+## Overview
+
+The Live Tail feature gives you the ability to see all your log events in near real time from anywhere in your infrastructure. It displays logs as soon as they get out of the [Pipeline section][1] and before [their indexation][2] by Datadog, hence: 
+
+1. All logs ingested by Datadog are displayed. ([The dynamic volume control][2] is not applied.)
+2. Displayed logs have been processed.
+3. The stream can be paused.
+4. You can't go back in the past.
+
+This feature allows you for instance to check if a process has correctly started or if a new deployment went smoothly.
+
+## Live tail view
+
+Choose the `Live Tail` option in the time range selector to switch to the live tail view:
+
+{{< img src="logs/live_tail/live_tail_time_selector.png" alt="Live tail time selector" responsive="true" popup="true">}}
+
+The number of received events per second is displayed at the top left as well as the sampling rate (since a stream of thousands of logs per seconds is not human readable, high throughput log stream are sampled).
```

Comment:
The number of received events per second is displayed at the top left as well as the sampling rate (since a stream of thousands of logs per seconds is not human readable, high throughput log streams are sampled).